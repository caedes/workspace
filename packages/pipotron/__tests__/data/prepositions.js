import prepositions from '../../src/data/prepositions';

describe('prepositions', () => {
  it('returns the prepositions', () => {
    expect(prepositions).toMatchSnapshot();
  });
});
