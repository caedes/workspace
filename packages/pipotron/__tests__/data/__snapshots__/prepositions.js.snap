// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prepositions returns the prepositions 1`] = `
Array [
  "Avec",
  "Considérant",
  "Où que nous mène",
  "Eu égard à",
  "Vu",
  "En ce qui concerne",
  "Dans le cas particulier de",
  "Quelle que soit",
  "Du fait de",
  "Tant que durera",
  "Quoi qu'on dise concernant",
  "Nonobstant",
  "Compte tenu de",
  "Malgré",
  "Pour réagir face à",
  "Afin de circonvenir à",
  "Dans le but de pallier à",
  "Si vous voulez mon avis concernant",
]
`;
